media/js/base/site.js

```diff
@@ -54,6 +54,25 @@
             return match ? match[1].replace('_', '.') : undefined;
         },
 
+        // Madness from https://docs.microsoft.com/microsoft-edge/web-platform/how-to-detect-win11
+        getWindowsVersionCH: function (version) {
```

Comment:
This'll sound pendatic, but... does `CH` mean `ClientHint`? If so, worth spelling out? 
I had a moment where I thought it was a Swiss thing before realising it's more likely client hints